collapsiblecalendarview2/src/main/res/values/strings.xml

```diff
@@ -8,4 +8,7 @@
     <string name="friday">FRI</string>
     <string name="saturday">SAT</string>
     <string name="sunday">SUN</string>
+    <string name="content_description_eiv_arrow">expand or collapse calendar</string>
+    <string name="content_description_rl_options_button">options menu button</string>
+    <string name="content_description_rl_today_button">select today button</string>
```

Comment:
Since these content descriptions are going to be used for views that are not traditional buttons, we need to make sure that the screen reader does not read out something like "Expand calendar button ExpandIconView" since TalkBack will read out the view type by default. If so, we may need to redo some layouts.

Reply:
i have used talkback to test and it does not do this

Reply:
i have used talkback to test and it does not do this